src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/coreperson/CorePersonEventListener.kt

```diff
@@ -0,0 +1,104 @@
+package uk.gov.justice.digital.hmpps.prisonerfromnomismigration.coreperson
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.listeners.EventFeatureSwitch
+import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.listeners.SQSMessage
+import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.listeners.asCompletableFuture
+import java.util.concurrent.CompletableFuture
+
+@Service
+class CorePersonEventListener(
+  private val service: CorePersonSynchronisationService,
+  private val objectMapper: ObjectMapper,
+  private val eventFeatureSwitch: EventFeatureSwitch,
+) {
+
+  private companion object {
+    private val log: Logger = LoggerFactory.getLogger(this::class.java)
+  }
+
+//  @SqsListener("eventcoreperson", factory = "hmppsQueueContainerFactoryProxy")
```

Comment:
deliberately comment out the actual listening at present.  Think we're a while away from getting syncing working but wanted to make sure that we've got all the events as there is a lead time on the requests.